src/db/tables/subscribers.ts

```diff
@@ -397,27 +392,17 @@ async function getSubscribersWaitingForMonthlyEmail(
           "\"monthly_monitor_report_at\" < NOW() - INTERVAL '1 month'",
         ),
     )
-    // ...and whose account is older than 1 month.
-    .andWhereRaw("\"created_at\" < NOW() - INTERVAL '1 month'");
-
-  if (Array.isArray(flag.allow_list) && flag.allow_list.length > 0) {
-    // If the feature flag has an allowlist, only send to users on that list.
-    // The `.andWhereIn` alias doesn't exist:
-    // https://github.com/knex/knex/issues/1881#issuecomment-275433906
-    query = query.whereIn("primary_email", flag.allow_list);
-  }
-
-  if (options.plusOnly) {
```

Comment:
This removes the flag check, and always limits this to Plus users (rather than accepting that as an option).